src/app/_components/RecentBlocks/__tests__/formatTimestamp.test.ts

```diff
@@ -0,0 +1,9 @@
+import { formatTimestamp } from '../utils';
+
+describe('formatTimestamp', () => {
```

Comment:
how does the function perform with other inputs besides a number?

Reply:
The function is typed `formatTimestamp(timestamp: number)` so it shouldn't accept any inputs aside from numbers.